.vscode/c_cpp_properties.json

```diff
@@ -6,7 +6,7 @@
             "compilerPath": "/usr/bin/gcc",
             "includePath": [
                 "${default}",
-                "${workspaceFolder}/sysroot/usr/include/kernel/",
+                "${workspaceFolder}/kernel/",
```

Comment:
So can no longer use the sysroot include directory?

Reply:
I removed it since we don't have anything in it now. I could add it back, but I figured we were going to reorganize it / add things as necessary when the time arrives.